desktop/core/src/desktop/static/desktop/locales/en/translation.json

```diff
@@ -277,6 +277,15 @@
   "Memory": "Memory",
   "Message": "Message",
   "Metrics": "Metrics",
+  "Active Requests": "Active Requests",
+  "Active Users": "Active Users",
+  "Daemon Threads": "Daemon Threads",
+  "Number of Queries": "Number of Queries",
```

Comment:
My suggestion was to use the keys here directly and drop the label mapping file completely.
E.g. 

` "queries.number": "Number of Queries",`